src/repo/zif_abapgit_repo_pre_filter.intf.abap

```diff
@@ -0,0 +1,42 @@
+INTERFACE zif_abapgit_repo_pre_filter
+  PUBLIC .
+
+  TYPES ty_trobj_name TYPE c LENGTH 120.    "Standard TROBJ_NAME
+  TYPES ty_trrngtrkor_tt TYPE RANGE OF trkorr.
+  TYPES ty_trrngtrkor TYPE LINE OF ty_trrngtrkor_tt.
+  TYPES ty_msg TYPE c LENGTH 100.
+  TYPES: BEGIN OF ty_e071_filter,
+           pgmid    TYPE tadir-pgmid,
+           object   TYPE tadir-object,
+           obj_name TYPE ty_trobj_name,
+         END OF ty_e071_filter,
+         ty_e071_filter_tt TYPE STANDARD TABLE OF ty_e071_filter,
+         ty_file_filter_tt TYPE RANGE OF string,
+         ty_file_filter    TYPE LINE OF ty_file_filter_tt.
```

Comment:
these types suggest that all filters are transport relevant, 
 see scenarios in https://github.com/abapGit/abapGit/issues/5054#issuecomment-951079434, we also need a filter that is "changed today" ie. not related to transports.